web/src/components/forms/OpsgenieDestinationForm/OpsGenieDestinationForm.test.tsx

```diff
@@ -68,12 +72,13 @@ describe('OpsGenieDestinationForm', () => {
       <OpsgenieDestinationForm onSubmit={() => {}} initialValues={emptyInitialValues} />
     );
     const displayNameField = getByLabelText('* Display Name');
+    const serviceRegion = getByLabelText('Service Region', { selector: 'input' });
```

Comment:
Have you tried if this works?

P.S. That applies to all similar cases found below

```suggestion
    const serviceRegion = getByLabelText('Service Region');
```

Reply:
Doesn't work